.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/GitVersionTfsTask/BuildScripts"]
```

Comment:
As already mentioned instead of a submodule this should be included by npm (see https://github.com/geeklearningio/gl-vsts-tasks-build-scripts/issues/1).

The whole build pipeline needs generally a little bit more work. It needs to be integrated into the existing build pipeline.